mk/config.mk

```diff
@@ -251,12 +251,19 @@ CFG_WITH_USER_TA ?= y
 # By default, in-tree TAs are built using the first architecture specified in
 # $(ta-targets).
 
+# When enabled Trusted Applications are compiled as position-independent
+# executables
+CFG_TA_PIE ?= y
+
 # Address Space Layout Randomization for user-mode Trusted Applications
 #
 # When this flag is enabled, the ELF loader will introduce a random offset
 # when mapping the application in user space. ASLR makes the exploitation of
 # memory corruption vulnerabilities more difficult.
-CFG_TA_ASLR ?= y
+# The flag is enabled only if CFG_TA_PIE is enabled.
```

Comment:
"The flag **~~is~~ can be** enabled only if CFG_TA_PIE is enabled."

Reply:
Fixed